src/main/java/com/checkmarx/flow/controller/GitLabController.java

```diff
@@ -150,7 +148,13 @@ public String getTest() {
                     .filter(filter)
                     .build();
 
+            String mergeEndpoint = properties.getConfigApiUrl(request).concat(GitLabService.MERGE_NOTE_PATH);
+            mergeEndpoint = mergeEndpoint.replace("{id}", proj.getId().toString());
+            mergeEndpoint = mergeEndpoint.replace("{iid}", objectAttributes.getIid().toString());
+            request.setMergeNoteUri(mergeEndpoint);
```

Comment:
It's better to extract this code into a method.

Reply:
Done